### ISSUE TITLE
6.58 Opcodes after hotfix

### ISSUE THREAD

**Issue by lwins-lights** [Soreepeong/XivAlexander#696]
**Note: ** I did not double check its validity and it does not work if you hit more than 8 targets.
```
{
	"C2S_ActionRequest": "0x01d5",
	"C2S_ActionRequestGroundTargeted": "0x0260",
	"Common_UseOodleTcp": true,
	"PatchCode": [],
	"S2C_ActionEffect01": "0x01ca",
	"S2C_ActionEffect08": "0x031f",
	"S2C_ActionEffect16": "0x000f",
	"S2C_ActionEffect24": "0x000f",
	"S2C_ActionEffect32": "0x000f",
	"S2C_ActorCast": "0x0391",
	"S2C_ActorControl": "0x0333",
	"S2C_ActorControlSelf": "0x0111",
	"Server_IpRange": "0.0.0.0/0",
	"Server_PortRange": "1-65535"
}
```

**Comment by TofuOrganic** [comment]
I got 
{
	"C2S_ActionRequest": "0x01d5",
	"C2S_ActionRequestGroundTargeted": "0x01d5",
	"Common_UseOodleTcp": true,
	"PatchCode": [],
	"S2C_ActionEffect01": "0x01ca",
	"S2C_ActionEffect08": "0x031f",
	"S2C_ActionEffect16": "0x02b2",
	"S2C_ActionEffect24": "0x0101",
	"S2C_ActionEffect32": "0x01f6",
	"S2C_ActorCast": "0x0391",
	"S2C_ActorControl": "0x0333",
	"S2C_ActorControlSelf": "0x0111",
	"Server_IpRange": "0.0.0.0/0",
	"Server_PortRange": "1-65535"
}

**Comment by lwins-lights** [comment]
@TofuOrganic Seems like we need to figure out `C2S_ActionRequestGroundTargeted`

**Comment by mihokaenbyou** [comment]
"C2S_ActionRequestGroundTargeted": "0x0260"

should be:

{
	"C2S_ActionRequest": "0x01d5",
	"C2S_ActionRequestGroundTargeted": "0x0260",
	"Common_UseOodleTcp": true,
	"PatchCode": [],
	"S2C_ActionEffect01": "0x01ca",
	"S2C_ActionEffect08": "0x031f",
	"S2C_ActionEffect16": "0x02b2",
	"S2C_ActionEffect24": "0x0101",
	"S2C_ActionEffect32": "0x01f6",
	"S2C_ActorCast": "0x0391",
	"S2C_ActorControl": "0x0333",
	"S2C_ActorControlSelf": "0x0111",
	"Server_IpRange": "0.0.0.0/0",
	"Server_PortRange": "1-65535"
}

**Comment by TofuOrganic** [comment]
Yeah that's correct must've looked at the same one by mistake lol
